include/nbl/builtin/hlsl/bxdf/transmission/oren_nayar.hlsl

```diff
@@ -61,51 +69,61 @@ struct SOrenNayar
         scalar_type cos_phi_sin_theta = max<scalar_type>(VdotL - maxNdotL * maxNdotV, 0.0);
         return (AB.x + AB.y * cos_phi_sin_theta * C);
     }
+    template<typename Query>
+    spectral_type __eval(NBL_CONST_REF_ARG(Query) query, NBL_CONST_REF_ARG(sample_type) _sample, NBL_CONST_REF_ARG(isotropic_interaction_type) interaction)
+    {
+        scalar_type NdotL = _sample.getNdotL(_clamp);
+        return hlsl::promote<spectral_type>(NdotL * numbers::inv_pi<scalar_type> * 0.5 * __rec_pi_factored_out_wo_clamps(query.getVdotL(), NdotL, interaction.getNdotV(_clamp)));
```

Comment:
why do we still keep `__rec_pi_factored_out_wo_clamps` as standalone and not fold it into `__eval` ?

Reply:
esp that `__rec_pi_factored_out_wo_clamps` has completely wrongly named arguments (`max` for things are `clamp`)